---
layout: page
title: Publications
---

<ol reversed>

<li>  
Cao, S.*, Wang, J. R.*, <b>Ji, S.</b>*, Yang, P., Dai, Y., Guo, S., et al. Estimation of tumor cell total mRNA expression in 15 cancer types predicts disease progression. Nat. Biotechnol. 1–10 (2022). [<a href="https://www.nature.com/articles/s41587-022-01342-x">read</a>] <br> *: contributed equally. 
</li>

<li> 
<b>Ji, S.</b>, Montierth, M. D. . & Wang, W. MuSE: A Novel Approach to Mutation Calling with Sample-Specific Error Modeling. Variant Calling. 21–27 (Springer, 2022). [<a href="https://link.springer.com/protocol/10.1007/978-1-0716-2293-3_2">read</a>]
</li>

<li> 
Maniakas, A., Henderson, Y. C., Hei, H., Peng, S., Chen, Y., Jiang, Y., <b>Ji, S.</b>, et al. Novel Anaplastic Thyroid Cancer PDXs and Cell Lines: Expanding Preclinical Models of Genetic Diversity. J. Clin. Endocrinol. Metab. 106, E4652–E4665 (2021). [<a href="https://academic.oup.com/jcem/article/106/11/e4652/6306517">read</a>]
</li>

<li> 
Jiang, Y., Yu, K., <b>Ji, S.</b>, Shin, S. J., Cao, S., Montierth, M. D., et al. CliP: subclonal architecture reconstruction of cancer cells in DNA sequencing data using a penalized likelihood model. bioRxiv 2021.03.31.437383 (2021) doi:10.1101/2021.03.31.437383. [<a href="https://www.biorxiv.org/content/10.1101/2021.03.31.437383.full.pdf">read pre-print</a>]
</li>

<li> 
<b>Ji, S.</b> Improving protein structure prediction using amino acid contact & distance prediction. (University of Birmingham, 2019). [<a href="https://etheses.bham.ac.uk/id/eprint/9044/7/Ji2019PhD.pdf">read</a>] 
</li>

<li> 
<b>Ji, S.</b>,  Oruç, T., Mead, L., Rehman, M. F., Thomas, C. M., Butterworth, S., et al. DeepCDpred: Inter-residue distance and contact prediction for improved prediction of protein structure. PLoS One 14, e0205214 (2019). [<a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0205214">read</a>]
</li>

<li> 
Rajasekar, K. V., <b>Ji, S.</b>, Coulthard, R. J., Ride, J. P., Reynolds, G. L., Winn, P. J., et al. Structure of SPH (self-incompatibility protein homologue) proteins: A widespread family of small, highly stable, secreted proteins. Biochem. J. 476, (2019). [<a href="https://portlandpress.com/biochemj/article/476/5/809/219538">read</a>]
</li>

<li> 
Huang, X., Sun, L., <b>Ji, S.</b>, Zhao, T., Zhang, W., Xu, J., et al. Kissing and nanotunneling mediate intermitochondrial communication in the heart. Proc. Natl. Acad. Sci. 110, 2846–2851 (2013). [<a href="https://www.pnas.org/doi/full/10.1073/pnas.1300741110">read</a>]
</li>

<li> 
<b>Ji, S.</b>, Ye, C., Li, F., Sun, W., Zhang, J., Huang, Y., et al. Automatic segmentation of white matter hyperintensities by an extended FitzHugh & Nagumo reaction diffusion model. J. Magn. Reson. Imaging 37, 343–350 (2013). [<a href="https://onlinelibrary.wiley.com/doi/pdf/10.1002/jmri.23836">read</a>]
</li>

<li> 
Li, B., Dong, L., Chen, B. and <b>Ji, S.</b>, Cai, W., Wang, Y., Zhang, J., Zhang, Z., Wang, X. and Fang, J. Turbo fast three-dimensional carotid artery black-blood MRI by combining three-dimensional MERGE sequence with compressed sensing. Magnetic resonance in medicine. 70, 1347-1352 (2013). [<a href="https://onlinelibrary.wiley.com/doi/pdf/10.1002/mrm.24579">read</a>]
</li>
</ol>

